pkg/cmd/cli/cmd/status.go

```diff
@@ -28,36 +29,60 @@ oc describe deploymentConfig, oc describe service).
 You can specify an output format of "-o dot" to have this command output the generated status
 graph in DOT format that is suitable for use by the "dot" command.`
 
-	statusExample = `  # Show an overview of the current project
-  $ %[1]s`
+	statusExample = `  # See an overview of the current project.
+  $ %[1]s
+
+  # Export the overview of the current project in an svg file.
+  $ %[1]s -o dot | dot -T svg -o project.svg
+
+  # See an overview of the current project including suggestions for any warnings or errors found.
+  $ %[1]s --suggest`
 )
 
-// NewCmdStatus implements the OpenShift cli status command
+// StatusOptions contains all the necessary options for the Openshift cli status command.
+type StatusOptions struct {
+	namespace string
+
+	outputFormat string
+	describer    *describe.ProjectStatusDescriber
+	out          io.Writer
+
+	suggest bool
+	hasArgs bool
+}
+
+// NewCmdStatus implements the OpenShift cli status command.
 func NewCmdStatus(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
-	outputFormat := ""
+	opts := &StatusOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "status",
+		Use:     fmt.Sprintf("%s [-o dot | --suggest ]", StatusRecommendedName),
 		Short:   "Show an overview of the current project",
 		Long:    statusLong,
 		Example: fmt.Sprintf(statusExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
-			if strings.ToLower(outputFormat) == "dot" {
-				cmdutil.CheckErr(RunGraph(f, out))
-				return
+			err := opts.Complete(f, args, out)
+			cmdutil.CheckErr(err)
+
+			if err := opts.Validate(); err != nil {
+				cmdutil.CheckErr(cmdutil.UsageError(cmd, err.Error()))
 			}
 
-			cmdutil.CheckErr(RunStatus(f, out))
+			err = opts.RunStatus()
+			cmdutil.CheckErr(err)
 		},
 	}
 
-	cmd.Flags().StringVarP(&outputFormat, "output", "o", outputFormat, "Output format. One of: dot.")
+	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", opts.outputFormat, "Output format. One of: dot.")
+	cmd.Flags().BoolVar(&opts.suggest, "suggest", opts.suggest, "Make suggestions for possible warnings/errors.")
```

Comment:
`--suggest` sounds good to me. Other ideas: `--hints`, `--tips`. We could use something more generic if we plan to add things other than suggestions in future, like `--verbose`.